### DISCUSSION TITLE
What the stable commit for curl 8.2.0? Thanks.

### DISCUSSION THREAD

**Discussion by TheBetterSolution** [curl/curl#11515] [Q&A] [answered]
Could you please help me for ensuring the stable commit for curl 8.2.0? Because I hope to update code to 8.2.0.
Thanks very much.

**Comment by bagder** [comment, marked as answer]
curl releases are generated and shipped as tarballs. Get them from https://curl.se/download.html or via https://github.com/curl/curl/releases

The releases are generated from the source code at a specific moment in time, and that moment gets tagged in git as `curl-[version]` with the dots replaced with underscores. Such as `curl-8_2_0`.
